ruby/vampires.rb

```diff
@@ -86,6 +86,7 @@
 if (name == "Drake Cula" || name == "Tu Fang" || name == "tu fang" || name == "drake cula")
```

Comment:
Great working parsing this out to find the first case to check!  Here is a nesting suggestion.

Lets consider order of operations here.  I understand that the instructions may be guiding you to just put in conditions as they are mentioned.  However, we could clean this up and make the file much more efficient by analyzing our conditions for optimal calls and runtime.

If the employee is named "Drake Cula" or "Tu Fang" then we know that they are definitely a vampire. If that's the case, is there any need to even run the rest of the code?

Consider nesting your conditions and creating a strong order of operations for optimized logic and runtime.

```
if name == "Drake Cula" || name == "Tu Fang"
   "definitely a vampire"
else
  if age_check
    do something
  elsif garlic_and_insurance_check
    do something
  else
    do something
  end
end
```